symengine/cwrapper.cpp

```diff
@@ -160,7 +160,7 @@ TypeID basic_get_type(const basic s)
     return static_cast<TypeID>(s->m->get_type_code());
 }
 
-void symbol_set(basic s, char *c)
+void symbol_set(basic s, const char *c)
```

Comment:
We currently don't use `const` in cases like this one:
```
void integer_set_si(basic s, long i)
```
which should probably be instead:
```
void integer_set_si(basic s, const long i)
```
So then in the same way, we could use
```
void symbol_set(basic s, const char * const c)
```